pyproject.toml

```
@@ -17,6 +17,8 @@ test = [
     "pytest-asyncio",
 ]
 
+[tool.setuptools_scm]
+
```

```suggestion
version_file = "pypush/_version.py"
```
Also, don't forget to add this into gitignore file